x-pack/plugins/observability/public/hooks/use_chart_theme.tsx

```diff
@@ -14,6 +14,12 @@ export function useChartTheme() {
 
   return {
     ...baseChartTheme,
+    chartMargins: {
```

Comment:
Do we want this for all charts? Might want to check the layout on other pages to make sure there's not too much/little padding.

Reply:
@smith 10px is the default pixel used by Elastic Charts ( https://github.com/elastic/elastic-charts/blob/master/src/utils/themes/theme_commons.ts#L31-L36). I had to manually add it to this hook because on this commit https://github.com/elastic/kibana/pull/83587/files#diff-dd09d7e5c206dd512d4f027e6a65987d891caea9ee6d9bc0debbaf4b2f21fa60R15 we changed to use the EUI theme as base theme, and the margins there is set to 0.

Reply:
Perhaps we should default to the EUI theme settings and put an `EuiSpacer` in between the panel title and chart? Then we won't necessarily always have to consider where the charts are placed in context of the layout?

Reply:
I like adding it here because we wouldn't have to be worried about it anymore, so a space won't be missed on new charts. WDYT @smith ?

Reply:
@cauemarcondes I'm fine with that as long as we double-check the chart/table heights for correctness everywhere.

Reply:
@smith nothing changes on the Serv. Overview page.

<img width="1289" alt="Screenshot 2020-12-14 at 16 43 59" src="https://user-images.githubusercontent.com/55978943/102102001-9c327380-3e2b-11eb-883d-a7d12857be83.png">